Gopkg.toml

```diff
@@ -6,10 +6,9 @@ required = [
   "k8s.io/code-generator/cmd/client-gen",
   "k8s.io/code-generator/cmd/lister-gen",
   "k8s.io/code-generator/cmd/informer-gen",
-  "k8s.io/code-generator/cmd/openapi-gen",
+  # "k8s.io/code-generator/cmd/openapi-gen",
```

Comment:
why is this commented out?

Reply:
`cmd/openapi-gen` is no longer part of the latest `k8s.io/code-generator` package. Keeping it here so it can be replaced/fixed later.

Reply:
We have git, we should drop this line. When I see something like this it is not clear whether it was intentional, something that did not get cleared up before PR submission, et al.

Reply:
Line removed